x-pack/plugins/apm/public/components/app/alerts_overview/alerts_overview_table.test.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { render, waitFor, act } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import React, { ReactNode } from 'react';
+import { MockApmPluginContextWrapper } from '../../../context/apm_plugin/mock_apm_plugin_context';
+import * as useApmParamsHooks from '../../../hooks/use_apm_params';
+import { createKibanaReactContext } from '@kbn/kibana-react-plugin/public';
+import { CoreStart } from '@kbn/core/public';
+import { AlertsOverview } from '.';
+
+const getAlertsStateTableMock = jest.fn();
```

Comment:
We've previously talked about using [`@storybook/testing-react`](https://storybook.js.org/addons/@storybook/testing-react) for UI unit tests (like this one) so we get a test and a storybook for a price of one.

WDYT using `@storybook/testing-react` here?